Core/Core/About/View/AboutView.swift

```diff
@@ -45,9 +46,10 @@ public struct AboutView: View {
                 }
                 Image("instructure", bundle: .core)
                     .padding(.vertical, 36)
-                    .accessibilityHidden(true)
+                    .accessibilityIdentifier("InstructureLogo")
```

Comment:
The instructure logo was hidden because it was considered a decorative image. If we make it focusable we should ensure that it has a valid label and not the autogenerated one. Also Siri pronounces the Instructure word wrong.

Reply:
Agreed. Hid it back.